daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/ExplicitDisclosureTest.scala

```diff
@@ -256,10 +250,14 @@ class ExplicitDisclosureTest extends ExplicitDisclosureTestMethods {
         }
       }
 
-      "wrongly typed contract disclosures are rejected" in {
+      "wrongly typed contract disclosures are not found" in {
```

Comment:
Contracts with incorrect types are now not found rather than being rejected

Reply:
:+1: